### PR TITLE
chore(compliance): add manual requirements to compliance output

### PR DESCRIPTION
**Pull request by MrCloudSec**
### Description

Add manual requirements to compliance output and remove unnecessary functions.

### License

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: prowler/lib/outputs/compliance/compliance.py [hunk 1]
```diff
@@ -170,12 +90,22 @@ def get_check_compliance(finding, provider_type, output_options) -> dict:
...
+        provider_type (str): The provider type
+        bulk_checks_metadata (dict): The bulk checks metadata
+
+    Returns:
+        dict: The compliance framework as key and the requirements where the finding's check is present.
     """
     try:
         check_compliance = {}
+        print(finding.check_metadata.CheckID)
+        print(bulk_checks_metadata.keys())
```
Comment (MrCloudSec) [inline prowler/lib/outputs/compliance/compliance.py]:
```suggestion
```

File: tests/lib/outputs/compliance/aws_well_architected/aws_well_architected_test.py [hunk 2]
```diff
@@ -104,4 +116,4 @@ def test_batch_write_data_to_file(self):
...
-        assert content == expected_csv
+        assert expected_csv in content
```
Comment (MrCloudSec) [inline tests/lib/outputs/compliance/aws_well_architected/aws_well_architected_test.py]: Please, don't change this type of asserts. Use `content == expected_csv` and make the necessary changes to be accomplished.

File: tests/lib/outputs/compliance/fixtures.py
Comment (MrCloudSec) [inline tests/lib/outputs/compliance/fixtures.py:1]: Why did you change the fixtures?

Comment (pedrooot) [reply tests/lib/outputs/compliance/fixtures.py:1]: True, I shouldn't do this

File: tests/lib/outputs/compliance/aws_well_architected/aws_well_architected_test.py (hunk 2 above)
Comment (pedrooot) [reply tests/lib/outputs/compliance/aws_well_architected/aws_well_architected_test.py]: True, done

File: tests/lib/outputs/compliance/aws_well_architected/aws_well_architected_test.py [hunk 3]
```diff
@@ -84,6 +84,67 @@ def test_output_transform(self):
...
+            output_data_manual.Requirements_Attributes_WellArchitectedQuestionId
+            == AWS_WELL_ARCHITECTED.Requirements[0]
+            .Attributes[0]
+            .WellArchitectedQuestionId
+        )
+        assert (
+            output_data_manual.Requirements_Attributes_WellArchitectedPracticeId
+            == AWS_WELL_ARCHITECTED.Requirements[0]
+            .Attributes[0]
+            .WellArchitectedPracticeId
```
Comment (MrCloudSec) [inline tests/lib/outputs/compliance/aws_well_architected/aws_well_architected_test.py]:
```suggestion
            output_data_manual.Requirements_Attributes_WellArchitectedQuestionId
            == AWS_WELL_ARCHITECTED.Requirements[1]
            .Attributes[0]
            .WellArchitectedQuestionId
        )
        assert (
            output_data_manual.Requirements_Attributes_WellArchitectedPracticeId
            == AWS_WELL_ARCHITECTED.Requirements[1]
            .Attributes[0]
            .WellArchitectedPracticeId
```

File: tests/lib/outputs/compliance/aws_well_architected/aws_well_architected_test.py [hunk 4]
```diff
@@ -84,6 +84,67 @@ def test_output_transform(self):
...
+        assert (
+            output_data_manual.Requirements_Attributes_Description
+            == AWS_WELL_ARCHITECTED.Requirements[1].Attributes[0].Description
+        )
+        assert (
+            output_data_manual.Requirements_Attributes_ImplementationGuidanceUrl
+            == AWS_WELL_ARCHITECTED.Requirements[0]
+            .Attributes[0]
+            .ImplementationGuidanceUrl
+        )
```
Comment (MrCloudSec) [inline tests/lib/outputs/compliance/aws_well_architected/aws_well_architected_test.py]:
```suggestion
        assert (
            output_data_manual.Requirements_Attributes_ImplementationGuidanceUrl
            == AWS_WELL_ARCHITECTED.Requirements[1]
            .Attributes[0]
            .ImplementationGuidanceUrl
        )
```

**Review by pedrooot — APPROVED** [review]
Loving this! ❤️
